packages/quantic/force-app/main/default/lwc/quanticRadioButtonsGroup/quanticRadioButtonsGroup.html

```diff
@@ -0,0 +1,57 @@
+<!-- sldsValidatorIgnore -->
+<template>
+  <fieldset class="slds-form-element">
+    <legend
+      class="slds-form-element__legend slds-form-element__label radio-buttons-group__legend"
+    >
+      {legend}
+    </legend>
+    <div class="slds-form-element__control">
+      <div class="slds-radio_button-group radio-buttons-group-container">
+        <template for:each={transformedOptions} for:item="option">
+          <span
+            key={option.value}
+            class={option.buttonCSSClass}
+            data-cy={option.value}
+            onmouseenter={option.mouseEnterHandler}
+            onmouseleave={option.mouseLeaveHandler}
+          >
+            <div class="radio-buttons-group__tooltip-container">
+              <template lwc:if={option.iconName}>
+                <span class="slds-grid slds-var-p-vertical_x-small">
+                  <lightning-icon
+                    class={option.iconCSSClass}
+                    icon-name={option.iconName}
+                    variant={option.iconVariant}
+                    size={iconSize}
+                  ></lightning-icon>
+                </span>
+              </template>
+              <input
+                type="radio"
+                class="radio-input--no-outline"
+                name={option.label}
+                id={option.indexId}
+                checked={option.isChecked}
+                value={option.value}
+                onchange={handleChange}
+                onfocus={handleFocus}
+                onblur={handleBlur}
+              />
```

Comment:
I believe the reason you have 1 of 1 in the screen reader instead of 1 of 4 is that the radio buttons are grouped together using the same name. If they have different names (here option.label) then html thinks that they are not related if I recall properly. 

https://www.w3.org/WAI/tutorials/forms/grouping/#radio-buttons

Reply:
Ohhh perhaps? Lemme test that.
I couldn't figure out the difference between the generated html from my component and the generated html from the salesforce radio-button-group component. Perhaps this is it

Reply:
eyyyy it works nice!
![image](https://github.com/coveo/ui-kit/assets/2488155/52cf3d7c-f39d-45d7-8717-d4e65ffa026f)